run_test.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash -x
+
+# Run sanity tests for sonic-mgmt-common.
```

Comment:
Is this to run tests locally on the development server? If yes please explore the `tools/test/translib-test.sh` script as well. You can execute gotests without compiling a test binary.